test/e2e/performance_test.go

```diff
@@ -0,0 +1,371 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+	"unsafe"
+
+	"golang.org/x/exp/rand"
+
+	v1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	seed                            uint64 = 0xA1E47 // Use a specific rand seed to make the generated workloads always same
+	performanceAppLabel                    = "antrea-performance-test"
+	performanceTestPodPrefix               = performanceAppLabel + "-pod"
+	podsConnectionNetworkPolicyName        = performanceTestPodPrefix + "pods.ingress"
+	workloadNetworkPolicyName              = performanceTestPodPrefix + "workloads.ingress"
+	abImage                                = "tangweiqiang/ab"
```

Comment:
what's the software required in this image? I can add the image to the antrea dockerhub repo.

Reply:
Thanks. I have pushed the `antrea:apache-bench` image to dockerhub (https://hub.docker.com/r/antrea/apache-bench), you can use it now.

I have also opened a PR for you to review: https://github.com/vmware-tanzu/antrea/pull/165